cmd/dashboard.go

```diff
@@ -121,7 +121,11 @@ func runDashboardCmd(cmd *cobra.Command, args []string) {
 	serverListen := dashboardserver.ListenType(viper.GetString(constants.ArgDashboardListen))
 	error_helpers.FailOnError(serverListen.IsValid())
 
-	if err := utils.IsPortBindable(int(serverPort)); err != nil {
+	serverHost := string(serverListen)
+	if serverListen == dashboardserver.ListenTypeNetwork {
+		serverHost = "0.0.0.0"
+	}
+	if err := utils.IsPortBindable(serverHost, int(serverPort)); err != nil {
```

Comment:
I did it that way because a port can be bindable on an interface but not on another one, isn't it?

Reply:
Since `steampipe` always needs to listen on `IPv4 localhost`, this was hard coded to `127.0.0.1`. 

Having said that, your diagnosis on false positives/negative also holds in that `IPv4 localhost:<port>` may be available, but the same port may not be available on all interfaces.

Maybe we should check this for all passed in interfaces (or discovered interfaces in case of `network`).

Reply:
Given the dashboard still only accept `local` or `network`, it should be ok with the latest version where `"127.0.0.1"` is passed for `local`,  `""` is passed otherwise, i.e. for `network`: 

https://github.com/turbot/steampipe/pull/3200/files#diff-25e18a1f1f4867209e7c192da8a7c57e9b985d94bacd2f1dad53655d0323c219R124-R128